android/src/main/java/com/dylanvann/fastimage/FastImageSource.java

```diff
@@ -99,9 +99,6 @@ public Object getSourceForLoad() {
     }
 
     public Uri getUri() {
-        if (mUri == null) {
```

Comment:
Any reason for removing this null check? Can `imageSource.getUri()` return null?

Reply:
in the last release  i did this, since i reverted this change so not needed